tests/pipeline/test_pipeline.py

```diff
@@ -287,3 +287,29 @@ def test_default_config(self, tmpdir, snapshot):
         assert error_topic == "resources-custom-config-app2-error"
 
         snapshot.assert_match(enriched_pipeline, "test-pipeline")
+
+    def test_atm_fraud_example(self, tmpdir, snapshot):
```

Comment:
Should we move this test to its own file? like `test_atm_fraud_pipelie.py` So we have a new snapshot from it?
I have this feeling that the massive snapshot is hard to debug and understand sometimes.

Reply:
good idea, can I do it in the followup #96? I have already refactored the test fixtures there